src/vs/base/browser/ui/selectBox/selectBoxCustom.ts

```diff
@@ -848,10 +848,13 @@ export class SelectBoxList implements ISelectBoxDelegate, IListVirtualDelegate<I
 
 		this.selectionDetailsPane.innerText = '';
 		const selectedIndex = e.indexes[0];
-		const description = this.options[selectedIndex].description;
-		const descriptionIsMarkdown = this.options[selectedIndex].descriptionIsMarkdown;
+		if (!selectedIndex) {
+			return;
+		}
+		const description = (typeof this.options[selectedIndex].description === 'string') ? this.options[selectedIndex].description : null;
```

Comment:
The error is:

> TypeError: Cannot read property 'description' of undefined

This means this code will still blow up, since `this.options[selectedIndex]` would be `undefined`, not `this.options[selectedIndex].description`.

Reply:
Checking `selectedIndex` and `this.options[selectedIndex]` simultaneously makes sense I think before even getting to addressing `description`.

Reply:
Well, the telemetry error only indicates that `this.options[selectedIndex]` is problematic, not anything else.